transcom/mymove
# Main b 21071 name column filtering on partial

## [B-21071](Latent: Filter columns to be more user friendly - Name)

## Summary
Improving filtering for the "Customer name" column for Moves queue & Payment Requests queue.
This will support searching partial text within both the first and last names (prior implementation was last name only)

## Integration PR
https://github.com/transcom/mymove/pull/13961

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code
1. Pull in branch and run the application with db migrations loaded.

### How to test

*Moves Queue*
1. Access the Office app
2. Login as a Services Counselor or TOO
3. Ensure the "Counseling Queue"/"Task Order Queue" tab is selected
4. Search a partial/whole name

*Payment Requests Queue*
1. Access the Officer app
2. Login as a TIO
3. Ensure the "Payment Request Queue" tab is selected
5. Search a partial/whole name

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
<img width="1520" alt="image" src="https://github.com/user-attachments/assets/c869e4d6-7788-4f2b-a15f-0ca553cdb150">

<img width="1520" alt="image" src="https://github.com/user-attachments/assets/70a4eb5f-18e6-4c68-8957-542dcdfb0975">
